spec/acceptance/groups_spec.rb

```diff
@@ -43,19 +43,21 @@
     header "Authorization", :auth_token
 
     example_request "POST /groups" do
-      expect(status).to eq 201
-
+      group = Group.first
       json_response = JSON.parse(response_body)
 
+      expect(status).to eq 201
+
       expect(json_response["data"]["attributes"]["name"]).
         to eq public_send(:name)
 
       expect(authenticated_user.groups.first.name).
         to eq public_send(:name)
 
-      group = Group.first
-
       expect(group.users.first).to eq authenticated_user
+
+      expect(authenticated_user.has_role? :moderator, group).
```

Comment:
Add parentheses to nested method call `authenticated_user.has_role? :moderator, group`.